projects/1.18/assets/untamedwilds/untamedwilds/lang/zh_cn.json

```
@@ -0,0 +1,726 @@
+  "entity.untamedwilds.bear_blind.sciname": "Speleoarctos cavernicola",
+  "entity.untamedwilds.bear_cave": "洞熊",
+  "entity.untamedwilds.bear_cave.sciname": "Ursus spelaeus",
+  "entity.untamedwilds.bear_panda": "大熊猫",
+  "entity.untamedwilds.bear_panda.sciname": "Ailuropoda melanoleuca",
+  "entity.untamedwilds.bear_polar": "北极熊",
+  "entity.untamedwilds.bear_polar.sciname": "Ursus maritimus",
+  "entity.untamedwilds.bear_spectacled": "眼镜熊",
+  "entity.untamedwilds.bear_spectacled.sciname": "Tremarctos ornatus",
+  "entity.untamedwilds.bear_sun": "马来熊",
+  "entity.untamedwilds.bear_sun.sciname": "Helarctos malayanus",
+  "minecraft.subtitles.polar_bear.ambient": "熊：哼叫",
+  "untamedwilds.subtitles.bear.ambient": "熊：咆哮",
+  "untamedwilds.subtitles.bear.warning": "熊：大吼",
+  "untamedwilds.subtitles.bear.hurt": "熊：受伤",
+  "untamedwilds.subtitles.bear.death": "熊：死亡",
+  "untamedwilds.subtitles.bear.baby.ambient": "熊崽：哼叫",
+  "untamedwilds.subtitles.bear.baby.cry": "熊崽：尖叫",
+
+  "item.untamedwilds.big_cat_spawn_egg": "大猫刷怪蛋",
+  "entity.untamedwilds.big_cat": "大猫",
+  "entity.untamedwilds.big_cat_jaguar": "美洲豹",
+  "entity.untamedwilds.big_cat_jaguar.sciname": "Panthera onca",
+  "entity.untamedwilds.big_cat_leopard": "金钱豹",
+  "entity.untamedwilds.big_cat_leopard.sciname": "Panthera pardus",
+  "entity.untamedwilds.big_cat_lion": "狮子",
+  "entity.untamedwilds.big_cat_lion.sciname": "Panthera leo",
+  "entity.untamedwilds.big_cat_puma": "美洲狮",
+  "entity.untamedwilds.big_cat_puma.sciname": "Puma concolor",
+  "entity.untamedwilds.big_cat_snow_leopard": "雪豹",
+  "entity.untamedwilds.big_cat_snow_leopard.sciname": "Panthera uncia",
+  "entity.untamedwilds.big_cat_tiger": "老虎",
+  "entity.untamedwilds.big_cat_tiger.sciname": "Panthera tigris",
+  "entity.untamedwilds.big_cat_cave_lion": "洞狮",
+  "entity.untamedwilds.big_cat_cave_lion.sciname": "Panthera spelaea",
+  "entity.untamedwilds.big_cat_marsupial_lion": "刽子手袋狮",
+  "entity.untamedwilds.big_cat_marsupial_lion.sciname": "Thylacoleo carnifex",
+  "entity.untamedwilds.big_cat_sabertooth": "毁灭刃齿虎",
+  "entity.untamedwilds.big_cat_sabertooth.sciname": "Smilodon populator",
+  "untamedwilds.subtitles.big_cat.ambient": "大猫：咆哮",
+  "untamedwilds.subtitles.big_cat.ambient_meow": "大猫：嗷呜",
+  "untamedwilds.subtitles.big_cat.hurt": "大猫：受伤",
+  "untamedwilds.subtitles.big_cat.death": "大猫：死亡",
+
+  "item.untamedwilds.tarantula_spawn_egg": "狼蛛刷怪蛋",
+  "entity.untamedwilds.tarantula": "狼蛛",
+  "item.untamedwilds.bottle_tarantula_black": "瓶装巴西黑丝绒",
+  "entity.untamedwilds.tarantula_black": "巴西黑丝绒",
+  "entity.untamedwilds.tarantula_black.sciname": "Grammostola pulchra",
+  "item.untamedwilds.bottle_tarantula_black_and_white": "瓶装巴西黑白脚",
+  "entity.untamedwilds.tarantula_black_and_white": "巴西黑白脚",
+  "entity.untamedwilds.tarantula_black_and_white.sciname": "Nhandu coloratovillosus",
+  "item.untamedwilds.bottle_tarantula_cobalt": "瓶装泰国金属蓝",
+  "entity.untamedwilds.tarantula_cobalt": "泰国金属蓝",

```

```suggestion
  "entity.untamedwilds.tarantula_cobalt": "金属蓝蛛",
```
https://baike.baidu.com/item/%E9%87%91%E5%B1%9E%E8%93%9D%E8%9B%9B/2250000?fr=ge_ala